### PR TITLE
Added pep8radius.

### PR DESCRIPTION
**Pull request by j-bennet**
## Description

Adds pep8 checks in travis using pep8radius.

Issue: https://github.com/dbcli/mycli/issues/392.

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,12 +8,15 @@ python:
 
 install:
   - pip install PyMySQL . pytest mock codecov pexpect behave
+  - pip install git+https://github.com/hayd/pep8radius.git
```
Comment (amjith) [inline .travis.yml]: Damn! Bleeding edge. 🔴

Comment (j-bennet) [inline .travis.yml]: That's because of `--error-status` option. It's not released to pypi yet.

Comment (amjith) [inline .travis.yml]: Not complaining. Just a remark. 😄

**Comment by amjith** [comment]
I've disabled the build job that runs when we update a branch. We should only be running the tests on PR.

**Pull request merged**